server/src/test/java/io/crate/integrationtests/LogicalReplicationITest.java

```diff
@@ -541,7 +543,106 @@ public void test_drop_subscribed_table_is_not_allowed() throws Exception {
         assertThrowsMatches(
             () ->  executeOnSubscriber("DROP TABLE t1"),
             OperationOnInaccessibleRelationException.class,
-            "The relation \"doc.t1\" doesn't allow DROP operations, because it is included in a logical replication."
+            "The relation \"doc.t1\" doesn't allow DROP operations, because it is included in a logical replication subscription."
+        );
+    }
+
+    @Test
+    public void test_rename_published_tables() throws Exception {
```

Comment:
Looks like this is a subset of ` test_alter_published_tables`. Can we remove it or make the other test case more specific?